config/software/datadog-gohai.rb

```diff
@@ -1,6 +1,9 @@
 name "datadog-gohai"
 default_version "last-stable"
 
+always_build true
```

Comment:
now that you've defined a `github` `source`  we don't need to "always build" gohai anymore